libmscore/property.cpp

```diff
@@ -660,6 +661,18 @@ QString propertyToString(Pid id, QVariant value, bool mscx)
             case P_TYPE::STRING:
             case P_TYPE::FRACTION:
                   return value.toString();
+            case P_TYPE::KEYMODE:
+                  switch (KeyMode(value.toInt())) {
+                        case KeyMode::NONE:
+                              return "none";
+                        case KeyMode::MAJOR:
+                              return "major";
+                        case KeyMode::MINOR:
+                              return "minor";
+                        default:
```

Comment:
This `switch` should probably include all the new `KeyMode` entries too (if this PR is still actual).

Reply:
oops

Reply:
fixed